docs/quickstarts/event-bus.rst

```diff
@@ -0,0 +1,4 @@
+Using the Open edX Event bus
+============================
+
+whole process of adding an event to a service and adding it to event bus
```

Comment:
Should this doc be deleted, or do you have a plan for it?

Reply:
Deleted!